src/gap_to_julia.jl

```diff
@@ -133,15 +133,15 @@ gap_to_julia(::Type{T}, x::GapInt) where {T<:Integer} = T(x)
 gap_to_julia(::Type{Rational{T}}, x::GapInt) where {T<:Integer} = Rational{T}(x)
 
 ## Floats
-gap_to_julia(::Type{T}, obj::GapObj) where {T<:AbstractFloat} = T(obj)
+gap_to_julia(::Type{T}, obj::GapObj) where {T<:AbstractFloat} = applicable(T, obj) ? T(obj) : T(Float64(obj))
```

Comment:
I am really uncomfortable with using `applicable` here, IMHO this results in difficult to understand and handle code.

I think a more appropriate fix is to just do
```suggestion
gap_to_julia(::Type{Float64}, obj::GapObj) = Float64(obj)
```
and anyone who wants to convert to stuff to `Float32` or `BigFloat` has to deal with it by themselves (e.g. for a GAP big int it's a bad idea to first convert to Float64 in that case).

Reply:
After looking again through the Float stuff, it seems that it caused no ambiguities, and it is fundamentally different to the removed String stuff, as it does not use a magic `convert`. So this doesn't need to be changed at all.

Reply:
OK, great! (Sorry, didn't see this comment before)